source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -907,6 +908,30 @@ message TransformationOutlineFunction {
 
 }
 
+message TransformationPermuteFunctionParameters {
+
+  // A transformation that, given a non-entry-point function taking n
+  // parameters and a permutation of the set [1, n]:
```

Comment:
Would it make things simpler to think of the permutation being on the set [0, n-1], since argument indexing starts from 0?

Reply:
Yes, I think so. As a matter of fact, I've been using function's return type id as a first element in the permutation to get rid of `+1` in transformation class everywhere (thus, `permutation[0] == 0` always). Fixed that in 19a2a10.